### PR TITLE
Added Bowen

### PR DESCRIPTION
**Pull request by bquan0**

### CODE REVIEW

**Comment by lewisgross1296** [comment]
@bquan0 Generally when you make a PR, you want to leave a description that explain the purpose so other reviewers know why it is being opened and what to look for when reviewing.

File: community/people/bq/Bowen_Quan.jpg:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (gonuke) [inline community/people/bq/Bowen_Quan.jpg:Zone.Identifier:1]: I don't think this file should be here.  Not sure where it comes from.

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for this addition @bquan0 - it looks like there is an extra file.

**Comment by gonuke** [comment]
Thanks @bquan0 - sorry this lingered so long!  You should feel free to nag me.

**Review by gonuke — APPROVED** [review]

**Pull request merged**